docs/LDAP.rst

```diff
@@ -372,7 +372,21 @@ you must add this configuration snippet under the “attributes” tag in the fi
 				   </attribute>
 		...
 			   </attributes>
-	
+
+These attributes can be retrieved in the User Profile with the Java API:
```

Comment:
```suggestion
These attributes can be retrieved in the Gatein User Profile with the Java API:
```

Reply:
Why adding **Gatein** (the product name, that only internal eXoers should know) ? If you want to be more specific, you should use *IDM* instead IMO.

Reply:
You're right.
I mean Portal User Profile, to make difference with Social User Profile
```suggestion
These attributes can be retrieved in the Portal User Profile with the Java API:
```

Reply:
I agree "Portal" is better than "Gatein". @boubaker ok for you for "Portal" ?

Reply:
I let you decide

Reply:
ok, let's go for Portal.